lib/ansible/module_utils/docker/common.py

```diff
@@ -664,6 +665,15 @@ def pull_image(self, name, tag="latest"):
     def report_warnings(self, result, warnings_key=None):
```

Comment:
What do you think about renaming `warnings_key` to `warning_keys` to make it more clear that it expects something iterable?

Reply:
`warning_keys` indicates to me that these are multiple keys, which can all be used to retrieve warnings. That's why I chose `warning_key`, because it is one key to the warnings (and the key can potentially be a path, that's why it is a list).

Reply:
Makes sense.